 83952    2 -rw-r--r--   1 sfe      other        1092 Jan 12 11:16 ./amd64/Makefile.orig
pkgbuild@hipster> diff -u ./amd64/Makefile.orig ./amd64/Makefile     
--- illumos-fusefs-Version-1.3.2/kernel/amd64/Makefile.orig       2018-01-12 11:16:07.000000000 +0000
+++ illumos-fusefs-Version-1.3.2/kernel/amd64/Makefile    2018-01-21 13:50:18.357505328 +0000
@@ -23,7 +23,8 @@
 # Copyright 2008 Sun Microsystems, Inc.  All rights reserved.
 # Use is subject to license terms.
 
-CFLAGS=-xmodel=kernel -m64
+#CFLAGS=-xmodel=kernel -m64
+CFLAGS= -ffreestanding -mcmodel=kernel -mno-red-zone -m64
 LINTFLAGS = -m64
 include ../Makefile.com
 
